src/WebJobs.Extensions/Utility.cs

```diff
@@ -23,5 +26,68 @@ public static bool IsDynamic
                 return string.Compare(value, DynamicSku, StringComparison.OrdinalIgnoreCase) == 0;
             }
         }
+
+        /// <summary>
+        /// Applies any additional binding data from the input value to the specified binding data.
+        /// This binding data then becomes available to the binding process (in the case of late bound bindings)
+        /// </summary>
+        internal static void ApplyBindingData(object value, Dictionary<string, object> bindingData)
```

Comment:
This code already lives in this project, as part of the http extension. However, since manual trigger won't have a binding contract, I think you won't need this.